examples/join-multiple-streams/join_multiple_streams.out

```diff
@@ -4,9 +4,9 @@ $ ballerina run join_multiple_streams.bal
 ballerina: initiating service(s) in 'join_multiple_streams.bal'
 ballerina: started HTTP/WS endpoint 0.0.0.0:9090
 
-# Then make below cUrl requests
+# Run the following curl requests. 
 $ curl -H "Content-Type: application/json" -X POST -d '{"name":"Teak","amount":1000.0}' http://localhost:9090/rawmaterial
 $ curl -H "Content-Type: application/json" -X POST -d '{"name":"Teak","amount":500.0}' http://localhost:9090/productionmaterial
 
-# After sending above two requests within same 10 seconds you could see below log get printed.
+# After running the curl commands, the following log is printed within 10 seconds. 
```

Comment:
Actually, if you sent cUrl requests within 10 seconds then log get printed. (It is not, log get printed within 10 seconds)

Reply:
Changed this with commit https://github.com/ballerina-platform/ballerina-examples/pull/696/commits/9deeea4429b469951eba612618460421619bcc9b. Please check and verify if it gives the correct meaning.